src/View/Form/EntityContext.php

```diff
@@ -629,7 +629,7 @@ protected function _getValidator(array $parts): Validator
      *
      * @param string[]|string|\Cake\Datasource\EntityInterface $parts Each one of the parts in a path for a field name
      * @param bool $fallback Whether or not to fallback to the last found table
-     *  when a non-existent field/property is being encountered.
+     *  when a nonexistent field/property is being encountered.
```

Comment:
Interesting, that my spelling recommendation given is the former
https://www.dict.cc/deutsch-englisch/nicht+existent.html

Reply:
Some dictionaries have it, but I was mainly cleaning up the spelling of existent and spacing. So, just decided to make them all consistently use the full word.

Reply:
Given https://english.stackexchange.com/questions/237113/nonexistent-non-existent-or-non-existent I would tend to agree with you :)
The prosecution rests your honor :P